### ISSUE TITLE
Observability docs missing?

### ISSUE THREAD

**Issue by tschut** [spring-cloud/spring-cloud-config#2246]
It appears the links in https://github.com/spring-cloud/spring-cloud-config/blob/main/docs/src/main/asciidoc/_observability.adoc are broken. The _observability.adoc file was committed here: https://github.com/spring-cloud/spring-cloud-config/pull/2081/files#diff-83346ace8fdc9ab4c6f56e5f11e32159f3df19d3bed9ce2b199095df539319c9.

I noticed when upgrading spring cloud to 2022.x the constructor parameters for `MultipleJGitEnvironmentRepository` have changed and I couldn't find any documentation for this.

Maybe I'm just looking in the wrong place?

**Comment by ryanjbaxter** [comment]
Thanks.  We missed including it in main docs.  That should be addressed here https://github.com/spring-cloud/spring-cloud-config/commit/69f11e8608f56c287049e1a6e41abd57b6f91198

What documentation are you looking for with regards to `MultipleJGitEnvironmentRepository`?

**Comment by tschut** [comment]
We have some tiny customizations on it which are implemented by subclassing `MultipleJGitEnvironmentRepository`. They were implemented by someone who left the project and to be honest I'm not sure we still need them. Anyway, when updating to spring cloud 2022.x that broke because `ObservationRegistry` was added to the constructor, so I started trying to find documentation on that.

TL;DR I'm not really missing any docs, just noticed something was missing and thought I'd point it out
